docs/administration-guide/operation-score.md

```diff
@@ -0,0 +1,178 @@
+
+- `opScores` - required map of operation scores. These are the operation score values.
+- `authScores` - required map of operation authorisation scores. These are the maximum scores allowed for a user with a given role. 
+- `scoreResolver` - some operations may require a custom way of calculating an associated score, therefore an implementation of the [`ScoreResolver`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/store/operation/resolver/ScoreResolver.html) interface may be required. 
+These map the operation class to its respective score resolver.
+There is a `DefaultScoreResolver` to which the custom implementation should delegate, in a manner specific to the new Operation.
+
+!!! example "Example OperationDeclarations.json for a NamedOperation"
+
+    ```json
+    {
+        "operations": [
+            {
+                "operation": "uk.gov.gchq.gaffer.operation.impl.ScoreOperationChain",
+                "handler": {
+                    "class": "uk.gov.gchq.gaffer.store.operation.handler.ScoreOperationChainHandler",
+                    "opScores": {
+                        "uk.gov.gchq.gaffer.operation.Operation": 2,
+                        "uk.gov.gchq.gaffer.operation.impl.generate.GenerateObjects": 0
+                    },
+                    "authScores": {
+                        "User": 4,
+                        "EnhancedUser": 10
+                    },
+                    "scoreResolvers": {
+                        "uk.gov.gchq.gaffer.named.operation.NamedOperation": {
+                            "class": "uk.gov.gchq.gaffer.store.operation.resolver.named.NamedOperationScoreResolver"
+                        }
+                    }
+                }
+            }
+        ]
+    }
+    ```
+
+For more examples of ScoreOperationChain refer to the [Misc Operations page in the Reference Guide](../reference/operations-guide/misc.md#scoreoperationchain).
+
+## Operation Chain Limiters
+
+`OperationChainLimiter` is a [GraphHook](../development-guide/project-structure/components/graph.md#graph-hooks) that checks a user is authorised to execute an operation chain based on that user's maximum chain score and the configured score value for each operation in the chain. 
+If you wish to use the ScoreOperationChain operation and this graph hook, then both need to have the same score configuration.
+
+To use the `OperationChainLimiter` GraphHook then you will need to configure that GraphHook to use the a Score Resolver interface. 
+The ScoreOperationChainDeclaration.json declares that a NamedOperation should be resolved using a [`NamedOperationScoreResolver`](https://gchq.github.io/Gaffer/uk/gov/gchq/gaffer/store/operation/resolver/named/NamedOperationScoreResolver.html).
+This will then allow you to have custom scores for each NamedOperation.
+
+!!! example "Example hook configuration"
+    
+    Configuration of the hook should look something like this:
+
+    ``` json
```

Comment:
Where would you put this configuration and how does it differ from the next example that goes in the operation declarations file? I'm not sure its quite clear which example you should follow out of the two.

Reply:
As this is a hook I believe it would go into the `graphConfig.json` file. It might be helpful to add a link to [the hooks section of the graph page which explains this](https://github.com/gchq/gaffer-doc/blob/v2docs/docs/development-guide/project-structure/components/graph.md#graph-hooks).